### PR TITLE
Performance tweaks

### PR DESCRIPTION
**Pull request by pastewka**
Addresses performance issues with analysis - #841 #847

### CODE REVIEW

**Comment by pastewka** [comment]
Hi @mcrot - here are the performance tweaks. They should get rid of most of the bottlenecks.

**Comment by pastewka** [comment]
Hi @mcrot - I've removed package updates from this PR. Note that some analysis results will need to be recomputed. (Those that use `PlotCardView`.)

File: config/settings/local.py [hunk 1]
```diff
@@ -61,9 +61,9 @@
 # django-debug-toolbar
 # ------------------------------------------------------------------------------
 # https://django-debug-toolbar.readthedocs.io/en/latest/installation.html#prerequisites
-INSTALLED_APPS += ['debug_toolbar']  # noqa F405
```
Comment (mcrot) [inline config/settings/local.py:64]: Is it intended that we disable the debug tool for development by default?
Or is this just a left-over from debugging?

**Review by mcrot — CHANGES_REQUESTED** [review]
Looks great! I'm looking forward to see this in production.

Two things:
- Shall we really disable the debug toolbar for development by default?
- The test for split dictionaries should be extended by the use of the supplementary dict.

**Review by mcrot — APPROVED** [review]
This can be merged. If there are no string reasons against, I would keep the debug toolbar.

File: config/settings/local.py (hunk 1 above)
Comment (pastewka) [reply config/settings/local.py:64]: I am readding it

**Review by mcrot — APPROVED** [review]

**Pull request merged**